pact/ledger.pact

```diff
@@ -251,6 +250,13 @@
       (emit-event (TOKEN id precision 0.0 policies uri)))
   )
 
+  (defun check-reserved:string (token-id:string)
+    " Checks token-id for reserved name and returns type if \
+    \ found or empty string. Reserved names start with a \
+    \ single char and colon, e.g. 't:foo', which would return 't' as type."
+    (let ((pfx (take 2 token-id)))
+      (if (= ":" (take -1 pfx)) (take 1 pfx) "")))
+
```

Comment:
Do we want to allow non-reserved token-ids? For example if the token-id doesn't contain a colon currently, `enforce-token-reserved` returns true?

Reply:
Good question, we left it open for custom token names and only reserved for tokens starting with`t:`, but we should take the same approach with the accounts and only allow `t:`